shared_libraries/orca_shared/database/use_cases/create_postgres_connection_uri.py

```diff
@@ -1,14 +1,17 @@
-import logging
 from typing import Any
 
+from aws_lambda_powertools import Logger
 from sqlalchemy.engine import URL
 
 from orca_shared.database.entities.postgres_connection_info import (
     PostgresConnectionInfo,
 )
 
+# Set AWS powertools logger
+Logger = Logger()
 
-def create_user_uri(db_connect_info: PostgresConnectionInfo, logger: logging.Logger) -> str:
+
+def create_user_uri(db_connect_info: PostgresConnectionInfo, logger: Logger) -> str:
```

Comment:
You should not use an instantiated class as a typehint, and the typehint should still be logging.Logger. Revert changes to this file.

Reply:
reverted